Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -645,6 +645,42 @@ static void logProcessPoolState(const WebProcessPool& pool)
 {
     m_weakObserver = adoptNS([[WKProcessPoolWeakObserver alloc] initWithWeakPtr:*this]);
 
+#if ENABLE(NOTIFYD_BLOCKING_IN_WEBCONTENT)
+    const char* messages[] = { "com.apple.WebKit.LibraryPathDiagnostics",
```

Comment:
I wonder if this should live in its own `#include` file, so that we could easily add new messages to it?

And are these identical to what's needed on iOS, or might different messages be needed?

Reply:
Would be nice to use `ASCIILiteral`

Reply:
Could this be `constexpr Vector<ASCIILiteral> messages { "com.apple.WebKit.LibraryPathDiagnostics", ...etc...`?

Reply:
I can't remember if that initialization works.

Reply:
If we're going to use a static vector, then we may as well store Strings inside the Vector since that'd what we end up needing. Otherwise, we keep converting from const char* (or ASCIILiteral) to String to send the IPC.